mdx-guide.md

```diff
@@ -213,6 +213,16 @@ We also have a specialized admonition for Qiskit Code Assistant prompt suggestio
 />
 ```
 
+### IBM Videos
+
+The `IBMVideo` component allows you to add embeded videos from [video.ibm.com](https://video.ibm.com).
+
+To use the component, you only need the video's `id` and a descriptive `title` that will be used for accessibility purposes:
+
+```mdx
+<IBMVideo id="134056207" title="This is an example"/>
+```
```

Comment:
Can you please instead replace the current videos section with this? We should solely be using IBM videos and not storing any videos in Git anymore.